docs/source/data_ingestion_and_preparation/transforms/sketcher.md

```diff
@@ -0,0 +1,12 @@
+## Sketcher
```

Comment:
Should this be called "sketch.md" - this seem to follow the pattern with the other transforms?

Reply:
Good question.  The access via DocSet is with a method called `sketch` (keeping with the verb forms), but the transform is called `Sketcher`.  It's not clear to me what's right, but I'll rename if you like.

Reply:
I think most customers will use this in the context of a docset, not the transform directly. Feels like we should then bias towards "sketch" in a section that is talking about how to use DocSet, IMO. The other sections also follow this pattern, it seems.